.github/workflows/release.yaml

```diff
@@ -71,3 +71,53 @@ jobs:
           asset_path: ./bin/kubectl-fdb
           asset_name: kubectl-fdb-${{ matrix.os }}-amd64
           asset_content_type: application/octet-stream
+  push_images:
+    name: Push Docker images
+    needs: create-release
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image:
+          - fdb-kubernetes-operator
+          - foundationdb-kubernetes-sidecar
+          - fdb-data-loader
+        include:
+          - image: fdb-kubernetes-operator
+            context: ./
+          - image: foundationdb-kubernetes-sidecar
+            context: ./foundationdb-kubernetes-sidecar
+          - image: fdb-data-loader
+            context: ./sample-apps/data-loader
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Get the version
+        id: get_tag
+        run: echo ::set-output name=TAG::${GITHUB_REF/refs\/tags\//}
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      # Will be deprecated -> https://github.com/docker/login-action#github-packages-docker-registry
+      # but GitHub Container Registry is still in beta and only works with a personal access token (PAT)
+      # See: https://docs.github.com/en/packages/guides/migrating-to-github-container-registry-for-docker-images
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: docker.pkg.github.com
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build and push to registry
+        uses: docker/build-push-action@v2
+        with:
+          context: ${{ matrix.context }}
+          # todo adjust to FoundationDB
```

Comment:
What does "adjust to FoundationDB" mean here?

Reply:
This was an old comment, I was not sure if I could use the context in all places.